--- a/src/libOpenImageIO/CMakeLists2.txt	Wed Oct 25 02:04:23 2017
+++ b/src/libOpenImageIO/CMakeLists.txt	Wed Oct 25 01:57:04 2017
@@ -151,7 +151,9 @@
     target_link_libraries (OpenImageIO psapi.lib)
 endif ()
 
-add_dependencies (OpenImageIO "${VISIBILITY_MAP_FILE}")
+if (VISIBILITY_MAP_FILE)
+    add_dependencies (OpenImageIO "${VISIBILITY_MAP_FILE}")
+endif ()
 
 if (USE_EXTERNAL_PUGIXML)
     target_link_libraries (OpenImageIO ${PUGIXML_LIBRARIES})
